py4cast/datasets/titan/__init__.py

```diff
@@ -20,19 +18,6 @@
 import xarray as xr
 from joblib import Parallel, delayed
 from skimage.transform import resize
-
-# Cyeccodes is a Cython module from Météo-France to read grib faster than xarray+cfgrib
-try:
-    from cyeccodes import nested_dd_iterator
-    from cyeccodes.eccodes import get_multi_messages_from_file
-
-    use_cyeccodes = True
-except ImportError:
-    warnings.warn(
-        f"Could not import Cyeccodes, switching to xarray+cfgrib. {traceback.format_exc()}"
-    )
-    use_cyeccodes = False
-
```

Comment:
Do we still need to download and compile eccodes in the Dockerfile btw ? I would say no if we delete that, so let's also delete that other bit in the Dockerfile.

Reply:
cfgrib requires the eccode Python package, right? If the user does not use conda, he has to compile eccodes. Could be an argument to use conda.